blog(www): Gazette August Edition (#16404)
gatsbyjs/gatsby: docs/blog/2019-08-06-gazette-august/index.md

---
title: "Gatsby Gazette: The August Edition"
date: 2019-08-06
author: Sidhartha Chatterjee
excerpt: "Welcome to the new and improved Gatsby Gazette"
tags:
  - gazette
---

Remember when I said I'll see you folks next week with a jam packed Gazette? It's been 3 months! 😱 If it's any consolation though, this one is a jam packed edition.

> ![Meme joking about how I said I'll be back in a week and readers are now skeletons because 3 months passed](./meme.jpg)

Reviewer: Note sure about the skeleton in this - I don't think I'd count it as family friendly so we should probably remove it.

Author: Good point! Replaced with more skeleton less meme 😛

First, an aside.

[Mac OS X Snow Leopard](https://en.wikipedia.org/wiki/Mac_OS_X_Snow_Leopard) (version 10.6) was the seventh major release of Mac OS X (now named macOS). Unlike those of previous versions, the goals of Snow Leopard were improved performance, greater efficiency and the reduction of its overall memory footprint. Addition of new end-user features was not a primary consideration: its name signified its goal to be a refinement of the previous OS X version, Leopard.

Similarly, earlier last month, we felt like we needed to take some time to slow down and clean up Gatsby after some of the big features that we've shipped recently ([themes](https://www.gatsbyjs.org/blog/2019-07-03-announcing-stable-release-gatsby-themes/), [per page manifest](https://www.gatsbyjs.org/blog/2019-06-12-performance-improvements-for-large-sites/)). Our goal was to squash bugs and issues that had happened as a result of moving fast and shipping.

We called this the [Snow Leopard](https://github.com/gatsbyjs/gatsby/projects/13) release.

We think this was a good idea because besides being the best React framework there is, we also want to be the most stable one.

> Looks like people think we're doing fine. 🙂

Reviewer: We should be proud of the work we're doing, but let's try to avoid being too self-congratulatory - Gatsby is not bug free yet 😅 I'd vote to remove this line and the following tweet as there are still plenty of unresolved issues and PRs around.

Author: Correcto! Removed self praise and added intent but kept tweet 😛

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">I have an opinionated starter for creating course sites built with <a href="https://twitter.com/gatsbyjs?ref_src=twsrc%5Etfw">@gatsbyjs</a> that I hadn&#39;t updated since January so I full well expected to spend a few hours fixing it after I updated all the deps.<br><br>Nope, just updated the packages and everything worked. Thanks Gatsby team!</p>&mdash; Brian Holt (@holtbt) <a href="https://twitter.com/holtbt/status/1158559782451609600?ref_src=twsrc%5Etfw">August 6, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

> 

Reviewer: Also highlight some of the work from the learning team? I've sent you a link.

Author: And done

### Jason Blitz Lengstorf

Jason has basically been living his life on livestream since the past month. He's done some incredible streams and has many more exciting ones coming up.

> My favourite of the past few ones was [Advanced GraphQL techniques in Gatsby](https://www.twitch.tv/videos/462900728) Mikhail Novikov.

Reviewer 2: This seems to be the start of the video.

```suggestion
My favourite of the past few ones was [Advanced GraphQL techniques in Gatsby](https://www.twitch.tv/videos/462874512) Mikhail Novikov.
```

Author: Aha, thank you!

Reviewer 3: Videos on Twitch are short lived (I think 2 weeks). Is there link for youtube video of that?

Reviewer 4: Correct that Twitch VODs are ephemeral. Given Jason's a Twitch affiliate, he actually gets 2 months before it is removed, but I agree it would be better to use the YT link instead

I don't think he's uploaded this one yet on his YT channel (maybe due to the technical issues on Jason's end during the stream)

Author: Yeah, not uploaded to YouTube yet. We'll update this once it's up.

> 

Reviewer: Possibly mention some of the other topics that have been covered recently? Or link to the YouTube playlist?

Author: Done

### Theme Jam

To celebrate the launch of Gatsby Themes, we announced the [Gatsby Theme Jam](https://themejam.gatsbyjs.org/) last month.

The response was incredible. Seeing so many wonderful themes from all of you has been spectacular and we're afraid we might like _too_ many of them to pick one.

The team is currently hard at work judging and picking their favourites so results should be out very soon!

## Community Highlights

> Gatsby has always been a community effort. We get a lot of contributions every month. Want proof?

Reviewer: This section could be reversed? Start with thanks, then on to highlights, and finally this tweet. That places the most importance on the thanking, then goes straight in to what's changed, finally summarising with this section.

Let's put the emphasis on you instead of us:

```suggestion
Gatsby has always been a community effort. You make a huge number of contributions every month. In fact we recently had a record breaking month:
```

Author: Much better! Thank you

- [kellyvaughn](https://github.com/kellyvaughn) [added support for metafields to product nodes](https://github.com/gatsbyjs/gatsby/pull/16312) in `gatsby-source-shopify`
- [datakurre](https://github.com/datakurre) added a great [fix to ensure touchNode](https://github.com/gatsbyjs/gatsby/pull/15919) populates typeOwners
- [grantglidewell](https://github.com/grantglidewell) added [support](https://github.com/gatsbyjs/gatsby/pull/14630) for Gatsby [Preview](https://www.gatsbyjs.com/preview) for Drupal in `gatsby-source-drupal`
- [elken](https://github.com/elken) added [support for the Prism command-line](https://github.com/gatsbyjs/gatsby/pull/16170) plugin in `gatsby-remark-prismjs`
- [cephalization](https://github.com/cephalization) added a [great error check](https://github.com/gatsbyjs/gatsby/pull/16272) in Gatsby

Thank you for all your contributions, folks!

## Gatsby Site and Theme of the month

Time to show off some great stuff people have been building

### Site of the Month

The Gatsby site of the month is [www.fourpost.com](https://www.fourpost.com)

We love it because:

- It's so fast
- The illustrations are so very pretty

> - Uses Gatsby Preview

Reviewer: Link to Gatsby Preview :)

Author: Already linked to Preview once in blog post 🤷‍♂ But why not, adding

- Pulls in content from DatoCMS & EventBrite

> ### Theme of the Month

Reviewer: Maybe we should wait until after the Theme Jam results before we start doing a theme of the month?

Author: Very very good question

Reviewer 4: I'm siding with @m-allanson on this that I think we should hold off for the theme jam before pushing this. Afterwards I am full on for promoting themes

Author: Removed for now

Here is what [Dennis Brotzky](https://twitter.com/_brotzky) (Co-founder, [narative.co](https://www.narative.co/)) had to say:

> We’ve always enjoyed using Gatsby for our projects and when themes were announcement we knew it would be special. Novela is our first dive into themes and we’ve quickly grown to love it. It’s exciting to be part of the formation of a new way to build and share Gatsby sites.

## Community Maintainers Meeting

The Core Maintainers Meeting is now the **Community** Maintainers Meeting.

We've renamed the meeting to emphasise that this time is for the community. You can join the meeting whether you've contributed to Gatsby, use it regularly or just tried it out for the first time. Everyone is welcome!

We've also restructured the meeting with a new format that includes

- time for questions
- time for updates
- live pull request reviews

We think live pull request reviews are a great way to involve the community in not only contributing code but also helping us merge those contributions in. We hope these give you some insight into how we review contributions and help make [your first contribution](http://gatsby.dev/pair-programming) even easier.

> In case you're not comfortable asking questions on the call (first call with strangers can be intimidating, we understand), we have a form to submit questions/pull requests for live review. The form is at https://gatsby.dev/oss-maintainers-form.

Reviewer: Removed _first_, I've been in loads of these and haven't lost the fear :)

```suggestion
In case you're not comfortable asking questions on the call (a video call with strangers can be intimidating, we understand), we have a form to submit questions/pull requests for live review. The form is at https://gatsby.dev/oss-maintainers-form.
```

Author: Haha true!

Here's a nifty [add to calendar link](http://gatsby.dev/oss-maintainers) for the meeting.

Join us on the meeting every Wednesday at 8:30am (PDT). 🙌

## We're hiring

> If you like what you see and want to work on the best React framework there is, we're now hiring.

Reviewer: Mention we're looking for engineers? for both the core and cloud teams.

_the best React framework there is_

This is subjective (clearly I agree :D) but I think being a little bit humble will go a long way here.

Author: Done

Check out the open roles at https://www.gatsbyjs.com/careers/

> We're lovely people, we promise. 💜

Reviewer: Let's make this about you instead of us? "We'd love to work with you" or similar?

Author: Done

---
title: "Gatsby Gazette: The August Edition"
date: 2019-08-06
author: Sidhartha Chatterjee
excerpt: "Welcome to the new and improved Gatsby Gazette"
tags:
  - gazette
---

Remember when I said I'll see you folks next week with a jam packed Gazette? It's been 3 months! 😱 If it's any consolation though, this one _is_ a jam packed edition.

![Meme joking about how I said I'll be back in a week and readers are now skeletons because 3 months passed](./meme.jpg)

Welcome to the _new_ and _improved_ Gatsby Gazette. It's been a while. We have a lot to talk about.

This edition features our work on making Gatsby more stable, the Themes Jam, some solid documentation improvements, exciting livestreams featuring Jason _Blitz_ Lengstorf, highlights from the community and the revamped Core Maintainers meeting.

## Team Highlights

### Snow Leopard Release

> The past few weeks has been all about what we're calling the Snow Leopard release (kudos to [Dustin](https://twitter.com/schaudustin) for the name).

Reviewer 5: I don't really deserve any kudos :) Not a me original ™️

Author: Hard disagree. You deserve credit for naming this _and_ the idea which I'll add now.

We called this the [Snow Leopard](https://github.com/gatsbyjs/gatsby/projects/13) release.

We intend to keep doing this every now and then to keep Gatsby stable and ensure you have painless upgrades.

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">I have an opinionated starter for creating course sites built with <a href="https://twitter.com/gatsbyjs?ref_src=twsrc%5Etfw">@gatsbyjs</a> that I hadn&#39;t updated since January so I full well expected to spend a few hours fixing it after I updated all the deps.<br/><br/>Nope, just updated the packages and everything worked. Thanks Gatsby team!</p>&mdash; Brian Holt (@holtbt) <a href="https://twitter.com/holtbt/status/1158559782451609600?ref_src=twsrc%5Etfw">August 6, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

### WPGraphQL for Advanced Custom Fields

> [WPGraphQL for Advanced Custom Fields](https://www.wpgraphql.com/acf/) is now open source. [Jason Bahl](https://twitter.com/jasonbahl) recently joined our team and open sourcing this is a step forward in our commitment to making Gatsby + WordPress seamless.

Reviewer 5: Could you introduce this? Explain what it is and _then_ that it's free?

@jasonbahl will likely have a pithy 1-2 liner for the value of WPGraphQL!

Author: Done! Wrote it myself because timezones

In case that isn't your jam, there's also [Gatsby + WordPress](https://www.youtube.com/watch?v=DH7I1xRrbxs), [building an RSS feed-powered podcast site](https://www.youtube.com/watch?v=0hGlvyuQiKQ), [composing and styling Gatsby themes](https://www.youtube.com/watch?v=6Z4p-qjnKCQ), [Microfrontends](https://www.youtube.com/watch?v=0Ta-awtLZTs) and even a [livestream on livestreaming](https://www.youtube.com/watch?v=rgTugjTDYaE)!

Here's a convenient [playlist](https://www.youtube.com/playlist?list=PLz8Iz-Fnk_eTpvd49Sa77NiF8Uqq5Iykx) if you want to binge watch these.

### Theme Jam

Talking about jam, to celebrate the launch of Gatsby Themes, we announced the [Gatsby Theme Jam](https://themejam.gatsbyjs.org/) last month.

The response was incredible. Seeing so many wonderful themes from all of you has been spectacular and we're afraid we might like _too_ many of them to pick one.

The team is currently hard at work judging and picking their favourites so results should be out very soon!

## Community Highlights

> Firstly, thank you for all your contributions, folks! Every single contribution to Gatsby (however small or big) makes it better. Some of the community highlights from the last month include:

Reviewer 5: ```suggestion
Firstly, thank you for all your contributions, folks! Every single contribution to Gatsby makes it better and every single contribution _matters_. Some of the community highlights from the last month include:
```

> - [lannonbr](https://github.com/lannonbr) [built a Table of Contents component](https://github.com/gatsbyjs/gatsby/pull/15251) that is live on our documentation and is great when reading a lengthy documentation page

Reviewer 5: Comment/nit should we use names here?

Author: Good idea! Handles work in GitHub land but names are better on the blog post for sure. Done.

- [kellyvaughn](https://github.com/kellyvaughn) [added support for metafields to product nodes](https://github.com/gatsbyjs/gatsby/pull/16312) in `gatsby-source-shopify`
- [datakurre](https://github.com/datakurre) added a great [fix to ensure touchNode](https://github.com/gatsbyjs/gatsby/pull/15919) populates typeOwners
- [grantglidewell](https://github.com/grantglidewell) added [support](https://github.com/gatsbyjs/gatsby/pull/14630) for Gatsby [Preview](https://www.gatsbyjs.com/preview) for Drupal in `gatsby-source-drupal`
- [elken](https://github.com/elken) added [support for the Prism command-line](https://github.com/gatsbyjs/gatsby/pull/16170) plugin in `gatsby-remark-prismjs`
- [cephalization](https://github.com/cephalization) added a [great error check](https://github.com/gatsbyjs/gatsby/pull/16272) in Gatsby

> Gatsby has always been a community effort. You make a _lot_ of contributions. In fact, we made a brand new record last month:

Reviewer 5: ```suggestion
Gatsby has always been a community effort. You make a _lot_ of contributions. In fact, we achieved a new record in merged PRs last month:
```

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">We&#39;ve merged in 500 PRs into <a href="https://twitter.com/gatsbyjs?ref_src=twsrc%5Etfw">@gatsbyjs</a> over the past month! 😱<br/><br/>Thank you for your contributions, folks! <a href="https://t.co/dsU85CzuGX">pic.twitter.com/dsU85CzuGX</a></p>&mdash; Sid (@chatsidhartha) <a href="https://twitter.com/chatsidhartha/status/1156334013247737856?ref_src=twsrc%5Etfw">July 30, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

> ## Gatsby Site of the month

Reviewer 5: Excellent idea!!

Time to show off some great stuff people have been building.

The Gatsby site of the month is [www.fourpost.com](https://www.fourpost.com)

We love it because:

> - It's so fast

Reviewer 5: Let's quantify fast!

Here's a webpagetest with a Lighthouse run. We should _generally_ validate these things before we assert it's fast. It is fast, but it's not _perfect_ nor "Gatsby fast". Few too many tracking scripts :/

https://www.webpagetest.org/result/190807_6R_436af0f9843ced4a0ffb946dc7f47c54/

Author: You're absolutely right. Removed and replaced with what made me feel like it's fast in the first place (image loading)

- The illustrations are so very pretty
- Uses [Gatsby Preview](https://www.gatsbyjs.com/preview)
- Pulls in content from DatoCMS & EventBrite

## Community Maintainers Meeting

The Core Maintainers Meeting is now the **Community** Maintainers Meeting.

> We've renamed the meeting to emphasise that this time is for the community. You can join the meeting whether you've contributed to Gatsby, use it regularly or just tried it out for the first time. Everyone is welcome!

Reviewer 5: nit, but US English 🙃 

```suggestion
We've renamed the meeting to emphasize that this time is for the community. You can join the meeting whether you've contributed to Gatsby, use it regularly, or are just trying it out for the first time. Everyone is welcome!
```

We've also restructured the meeting with a new format that includes

- time for questions
- time for updates
- live pull request reviews

> We think live pull request reviews are a great way to involve the community in not only contributing code but also helping us merge those contributions in. We hope these give you some insight into how we review contributions and help make [your first contribution](http://gatsby.dev/pair-programming) even easier.

Reviewer 5: ```suggestion
Live pull request reviews are a great way to involve the community in growing understanding of the PR process _and_ to also get user-impactful PRs merged! We hope these give you some insight into how we review contributions and help make [your first contribution](http://gatsby.dev/pair-programming) as smooth as possible.
```

> In case you're not comfortable asking questions on the call (a video call with strangers can be intimidating, we understand), we have a form to submit questions/pull requests for live review. The form is at https://gatsby.dev/oss-maintainers-form.

Reviewer 5: TODO: add this link!

Author: Already done before I added it here 😉

---
title: "Gatsby Gazette: The August Edition"
date: 2019-08-06
author: Sidhartha Chatterjee
excerpt: "Welcome to the new and improved Gatsby Gazette"
tags:
  - gazette
---

> Remember when I said I'll see you folks next week with a jam packed Gazette? It's been 3 months! 😱 If it's any consolation though, this one _is_ a jam packed edition.

Author: I _kinda_ like the self deprecating humour! 😛 Added the part about cadence though.

> ![Meme joking about how I said I'll be back in a week and readers are now skeletons because 3 months passed](./meme.jpg)

Author: Okay, removed!

## Team Highlights

### Snow Leopard Release

The past few weeks has been all about what we're calling the Snow Leopard release (kudos to [Dustin](https://twitter.com/schaudustin) for the name).

First, an aside.

[Snow Leopard](https://en.wikipedia.org/wiki/Mac_OS_X_Snow_Leopard) was the seventh major release of macOS. Unlike those of previous versions, the goals of Snow Leopard were improved performance, greater efficiency and the reduction of its overall memory footprint. Addition of new end-user features was not a primary consideration: its name signified its goal to be a refinement of the previous OS X version, Leopard.

Similarly, earlier last month, we felt like we needed to take some time to slow down and clean up Gatsby after some of the big features that we've shipped recently ([themes](https://www.gatsbyjs.org/blog/2019-07-03-announcing-stable-release-gatsby-themes/), [per page manifest](https://www.gatsbyjs.org/blog/2019-06-12-performance-improvements-for-large-sites/)). Our goal was to squash bugs and issues that had happened as a result of moving fast and shipping.

> We called this the [Snow Leopard](https://github.com/gatsbyjs/gatsby/projects/13) release.

Author: Yes, indeed. Removed and added link to project at the bottom.

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">I have an opinionated starter for creating course sites built with <a href="https://twitter.com/gatsbyjs?ref_src=twsrc%5Etfw">@gatsbyjs</a> that I hadn&#39;t updated since January so I full well expected to spend a few hours fixing it after I updated all the deps.<br/><br/>Nope, just updated the packages and everything worked. Thanks Gatsby team!</p>&mdash; Brian Holt (@holtbt) <a href="https://twitter.com/holtbt/status/1158559782451609600?ref_src=twsrc%5Etfw">August 6, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

### WPGraphQL for Advanced Custom Fields

[WPGraphQL for Advanced Custom Fields](https://www.wpgraphql.com/acf/) is now open source. [Jason Bahl](https://twitter.com/jasonbahl) recently joined our team and open sourcing this is a step forward in our commitment to making Gatsby + WordPress seamless.

### Learning

Our learning team has been hard at work on improving the documentation on [gatsbyjs.org](https://gatsbyjs.org)

Amongst many many other things, we've added

- Next and previous pagination links to guide users through the documentation

> - A floating Table of Contents on most pages

Author: Hahahaha, sorry I missed it! Adding 🤗
